### PR TITLE
In 140143341 add people search highlighting

### PR DESCRIPTION
**Pull request by brunoOSI**

### CODE REVIEW

File: app/assets/javascripts/components/common/AgeInfo.jsx [hunk 1]
```diff
@@ -1,11 +1,34 @@
 import React from 'react'
 import moment from 'moment'
+import sanitizeHtml from 'sanitize-html'
 
 const AgeInfo = ({dateOfBirth}) => {
```
Comment (aaronOSI) [inline app/assets/javascripts/components/common/AgeInfo.jsx]: Pretty hard to understand what's going on in AgeInfo, is it possible to refactor?

File: app/assets/javascripts/components/common/PersonSuggestion.jsx [hunk 2]
```diff
@@ -5,25 +5,39 @@ import Languages from 'components/common/LanguageInfo'
...
-    <div className='col-md-6'>
-      {address && <AddressInfo {...address} /> }
-      {phoneNumber && <PhoneNumberInfo {...phoneNumber} />}
-    </div>
-  </div>
-)
+const PersonSuggestion = ({firstName, lastName, middleName, nameSuffix, dateOfBirth, gender, languages, races, ethnicity, ssn, address, phoneNumber}) => {
+  const fullName = [firstName, middleName, lastName, NAME_SUFFIX[nameSuffix]].filter(Boolean).join(' ')
+  const sanitizedField = (field) => ({
+    dangerouslySetInnerHTML: {
```
Comment (aaronOSI) [inline app/assets/javascripts/components/common/PersonSuggestion.jsx]: With a name that includes the word `dangerously`, I have to ask... can this be refactored? Especially if it's just for the AgeInfo handling

**Comment by aaronOSI** [comment]
@brunoOSI left a few comments, leaving to your judgement, otherwise looks good

File: app/assets/javascripts/components/common/PersonSuggestion.jsx (hunk 2 above)
Comment (NateOSI) [inline app/assets/javascripts/components/common/PersonSuggestion.jsx]: This is how we set raw html in React.

**Pull request closed without merging**